packages/compass-saved-aggregations-queries/src/stores/delete-item.ts

```diff
@@ -76,6 +79,17 @@ export const deleteItemConfirm = (): ThunkAction<
     if (!item) {
       return;
     }
+
+    track(
+      item.type == 'aggregation'
+        ? 'Aggregation Deleted'
```

Comment:
Just remembered that in `pipelineStorage.js` we do tracking when we delete an aggregation. Should we move these to the respective classes?

Reply:
Ah, missed that somehow, thanks. I don't think we can do this from these stores for the same reasons I moved `Aggregation Opened` outside of the action into a different place. If you do it from store, then you need to pass `screen` information to the store also and this just leaks implementation details that the store shouldn't care about I think

Reply:
Done in 5111259cf7cfb3404c74958a9585af733972eb55 but please tell me if the reasoning makes sense to you, we can talk more about it and then change it if needed